src/components/button/index.tsx

```diff
@@ -153,7 +153,7 @@ export const AppButton = ({
         ) : (
           <AppSpinner
             size={buttonHeight - defaultPadding * 2}
-            color={sharedColors.black}
+            color={textColor as string}
```

Comment:
Make sure that AppSpinner accepts the correct color type or make it accept string and ColorValue